packages/webapp-libs/webapp-tenants/src/components/tenantSwitch/__tests__/tenantSwitch.component.spec.tsx

```diff
@@ -44,4 +52,108 @@ describe('TenantSwitch: Component', () => {
     expect(await screen.findByText(tenantName)).toBeInTheDocument();
     expect(screen.queryByTestId('tenant-settings-btn')).not.toBeInTheDocument();
   });
+
+  it('should not render organisation or invitation if not tenants', async () => {
+    const tenantName = 'Test Tenant';
+    const tenants = [tenantFactory({ name: tenantName, membership: { role: TenantUserRole.OWNER } })];
+    const apolloMocks = [fillCommonQueryWithUser(currentUserFactory({ tenants }))];
+    render(<Component />, { apolloMocks });
+    expect(await screen.findByText(tenantName)).toBeInTheDocument();
+    expect(screen.queryByText(/organizations/i)).not.toBeInTheDocument();
+    expect(screen.queryByText(/invitations/i)).not.toBeInTheDocument();
+  });
+
+  it('should handle invitation tenant click', async () => {
+    const currentTenantName = 'Personal name';
+    const tenantName = 'Test Tenant';
```

Comment:
Could you move these variables to upper scope so they won't be duplicated in each test?